build.gradle

```diff
@@ -1,40 +1,57 @@
+//version: ce57b88f4e61acd8b360c8e1777e707054d79b17
+/*
+DO NOT CHANGE THIS FILE!
+
+Also, you may replace this file at any time if there is an update available.
+Please check https://github.com/SinTh0r4s/ExampleMod1.7.10/blob/main/build.gradle for updates.
+ */
+
+
+import com.github.jengelman.gradle.plugins.shadow.tasks.ConfigureShadowRelocation
+
+import java.util.concurrent.TimeUnit
+
 buildscript {
     repositories {
-        mavenCentral()
-        jcenter()
         maven {
-            name = "gt"
-            url = "https://gregtech.overminddl1.com/"
+            name = "jitpack"
```

Comment:
jitpack should be lower in the list?

Reply:
The build script is loaded from the example mod repo. Please open an issue there

Reply:
I've mentioned this before.  And regardless of where it's from, I'm reviewing for the NEI repo, jitpack being that high isn't appropriate for building NEI.